vod/common.h

```diff
@@ -16,13 +16,19 @@
 #define vod_div_ceil(x, y) (((x) + (y) - 1) / (y))
 #define vod_array_entries(x) (sizeof(x) / sizeof(x[0]))
 
-#define vod_is_bit_set(mask, index) (((mask)[(index) >> 3] >> ((index) & 7)) & 1)
-#define vod_set_bit(mask, index) (mask)[(index) >> 3] |= 1 << ((index) & 7)
+// bit sets
+#define vod_number_of_bits(i) (sizeof(i) * 8)
```

Comment:
looking at this again, since we already decided we use uint64_t for the bitmask elements, and we're making this assumption in a few place, I think it would be cleaner to just replace all vod_number_of_bits(...) with '64'